packages/core/src/domain/session/sessionCookieStore.ts

```diff
@@ -155,3 +162,28 @@ function isExpiredState(session: SessionState) {
 function clearSession(options: CookieOptions) {
   setCookie(SESSION_COOKIE_NAME, '', 0, options)
 }
+
+const MAX_AUDIT_ENTRIES = 50
```

Comment:
Observed cases seems to happen at the beginning of the session, 50 should be enough to debug with impacting memory